projects/plugins/jetpack/modules/sso/class.jetpack-sso-user-admin.php

```diff
@@ -1,1016 +1,244 @@
 <?php // phpcs:ignore WordPress.Files.FileName.InvalidClassFileName
-use Automattic\Jetpack\Assets;
-use Automattic\Jetpack\Connection\Client;
-use Automattic\Jetpack\Connection\Manager;
-use Automattic\Jetpack\Roles;
-use Automattic\Jetpack\Status\Host;
-use Automattic\Jetpack\Tracking;
+/**
+ * Extra UI elements added to the User Menu for the SSO feature.
+ *
+ * @deprecated $$next-version$$ Use Automattic\Jetpack\Connection\Manager\SSO instead.
+ *
+ * phpcs:disable VariableAnalysis.CodeAnalysis.VariableAnalysis.UnusedVariable
+ *
+ * @package automattic/jetpack
+ */
 
 if ( ! class_exists( 'Jetpack_SSO_User_Admin' ) ) :
-	require_once JETPACK__PLUGIN_DIR . 'modules/sso/class.jetpack-sso-helpers.php';
 	/**
 	 * Jetpack sso user admin class.
+	 *
+	 * @deprecated $$next-version$$
 	 */
 	class Jetpack_SSO_User_Admin {
```

Comment:
I think we can remove private properties and methods from this class.

Reply:
Done in 7745ce88c3591c0c5a9c89d3787803cd01355aaa